src/less/documentation-swedbankpay.less

```
@@ -1852,6 +1854,16 @@
             height: fit-content;
         }
     }
+
+    #dropdown-doc {
+        .component-preview-content {
+            .dropdown {
+                .dropdown-menu {
+                    z-index: calc(@dropdown-z-index + 3);
+                }
+            }
+        }
+    }
 }
```

since you brought the "infinite nesting pitfall 🌌" topic yesterday: here is a perfect example where nested is _not_ needed, since we're only targeting 1 rule

```suggestion
#dropdown-doc .component-preview-content .dropdown .dropdown-menu {
    z-index: calc(@dropdown-z-index + 3);
}
```

it is not critical, since it is not bringing any bug, it will end up with the same syntax once the preprocessor has compiled it, but it is not necessary and therefore can be simplified